src/sections/community/Community.Suborgs.tsx

```diff
@@ -442,7 +456,7 @@ const BitByteDescription: React.FC<{ isMobile: boolean }> = ({ isMobile }) => (
 const OutreachTitle: React.FC<{ isMobile: boolean, open?:boolean, setSelected: (org: string) => void; glowClassName: string}> = ({ isMobile, open, setSelected, glowClassName }) => (
   <CommunitiesTitle
     org="Reach"
-    color="DarkGreen"
+    color="green"
```

Comment:
Outreach green is [`#469B68`](https://www.figma.com/design/I0x5ajcgNtgCLzzn4ep42j/Marketing-Materials---2024-25?node-id=71-133&t=KsXct7Umzfqbu7yl-4)